devel/tektoncd-cli/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/tektoncd/cli 0.10.0 v
+name                tektoncd-cli
+
+categories          devel
+license             Apache-2
+platforms           darwin
+supported_archs     x86_64 i386
+
+description         A CLI for interacting with Tekton
+
+long_description    The Tekton Pipelines cli project provides a CLI for \
+                    interacting with Tekton. Tekton is a powerful yet \
+                    flexible framework for building CI/CD systems on \
+                    Kubernetes. It lets you build, test, and deploy across \
+                    multiple cloud providers or on-premises systems by \
+                    abstracting away the underlying implementation details.
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+checksums           rmd160  74c692dcace7b499612798a80b8e88e95b5a0c97 \
+                    sha256  150344c4997db792594a8683da27fa66ef0ca70c7a69494496af90e98067a0b2 \
+                    size    6964051
+
+if {${build_arch} eq "x86_64"} {
+    set tektoncd_target amd64
+} else {
+    set tektoncd_target 386
+}
+
+post-extract {
+    # Modify the build target to only build for darwin.
+    reinplace -E "s/(\s*)(.*GOOS=\[\^d\]\[\^a\]\[\^r\].*${tektoncd_target})/\\1#\\2/g" Makefile
+}
```

Comment:
I would prefer a patchfile here instead of an (unreadable) `reinplace`

Reply:
OK, change made. Had been hoping to diminish the maintenance burden of patch files from version to version, but fair point.